src/client/pps/pps.proto

```diff
@@ -92,6 +92,7 @@ message PipelineInfo {
   google.protobuf.Timestamp created_at = 6;
   PipelineState state = 7;
   string recent_error = 8;
+  map<int32, int32> job_counts = 9;
```

Comment:
Good point, ridiculously enough proto won't let me use states as map keys... even though they compile to int32s. I guess maybe that wouldn't work in other languages?